src/test/scala/com/cognite/sdk/scala/v1/ClientTest.scala

```diff
@@ -5,27 +5,53 @@ import com.cognite.sdk.scala.common.{ApiKeyAuth, Auth, InvalidAuthentication, Sd
 
 class ClientTest extends SdkTest {
   "Client" should "fetch the project using login/status if necessary" in {
-    noException should be thrownBy new GenericClient("scala-sdk-test")(
+    noException should be thrownBy new GenericClient(
```

Comment:
This and the following lines should stay the same when we have a default value for `baseUri`